rakelib/artifacts.rake

```diff
@@ -90,15 +156,15 @@ namespace "artifact" do
     path = matches.first[gem_line_regex, 1]
 
     if path
-      Rake::Task["plugin:install-local-core-gem"].invoke("logstash-core", path)
+      Rake::Task["plugin:build-local-core-gem"].invoke("logstash-core", path)
     else
-      puts("[artifact:install-logstash-core] using logstash-core from Rubygems")
+      exit(1)
```

Comment:
it's just that at this point it should never happen, we should always have gem "logstash-core", :path => ...
if not there's something wrong